core/core/sustainability_labels/sustainability-labels-evaluation.csv

```diff
@@ -4,10 +4,14 @@ certificate:BIORE,bioRe,Fashion,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 certificate:BLUE_ANGEL,Blauer Engel,Fashion,93,86,0,81,100,100,0,24,40,55,0,0,0,0
 certificate:BLUE_ANGEL_LAPTOPS,Blauer Engel - Laptops,Electronics,73,77,87,0,0,100,40,71,0,0,0,0,0,0
 certificate:BLUE_ANGEL_LEATHER,Blaue Engel - Leder,Fashion,73,67,0,100,60,0,0,0,0,22,0,0,0,0
+certificate:BLUE_ANGEL_SMARTPHONES,Blaue Engel - Smartphones,Electronics/Smartphones,73,71,83,0,0,50,25,67,0,72,0,0,0,100
+certificate:BLUE_ANGEL_PRINTER,Blaue Engel - Printer,Electronics,73,88,50,0,0,50,25,67,0,50,0,0,0,100

```

Comment:
Is there a special reason for using the singular form here? Most of the others use the plural form and it would be great if we can work towards a standard here.

Reply:
Hmm, not really, I guess.
This is what I got from Lena, but I think I have some knowledge gaps here so it might be good that we talk in the following days so that I'll know which issues to raise before applying the new labels to our code?

Reply:
We decided offline to do the standardization as part of a separate PR.

Reply:
Noted in this issue
https://cloud.bht-berlin.de/index.php/apps/deck/#/board/1463/card/4457